.github/workflows/website-publish.yml

```diff
@@ -0,0 +1,40 @@
+name: Publish website
+on:
+  workflow_dispatch:
+
+permissions:
+  # required to retrieve AWS credentials
+  id-token: write
+  contents: write
+  packages: write
+  pull-requests: read
+
+jobs:
+  Publish-Website:
+    environment: Production
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: website
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: ${{ secrets.AWS_IAM_ROLE }}
+          role-session-name: integration-testing
+          aws-region: us-east-1
+
+      - name: Build docs
```

Comment:
Can we run and call Docker commands please? We are again duplicating things between the Makefile and here. Worst case, call "local" versions of the make commands and have the Makefile docker commands call the local commands inside a container.

Reply:
Not sure I fully understand you. We are running `make` commands here which internally runs within docker. If you are talking about the push/sync, I'll move them to be run via `make`.

Reply:
Yes, I would like the make commands to be called instead of having to replicate the commands in two places.

Reply:
Done!